occjava/src/BRepBuilderAPI.i

```diff
@@ -220,3 +222,12 @@ class BRepBuilderAPI_NurbsConvert : public BRepBuilderAPI_ModifyShape
 		const Standard_Boolean Copy = Standard_False) ;
 };
 
+class BRepOffsetAPI_ThruSections  : public BRepBuilderAPI_MakeShape
+{
+public:
+	BRepOffsetAPI_ThruSections(const Standard_Boolean isSolid = Standard_False, const Standard_Boolean ruled = Standard_False, const Standard_Real pres3d = 1.0e-06);
+	void AddWire (const TopoDS_Wire& wire) ;
```

Comment:
AddWire and CheckCompatibility should start with a lower case letter to match java coding conventions.